workers/social/lib/social/models/group/index.coffee

```diff
@@ -1501,8 +1532,22 @@ module.exports = class JGroup extends Module
 
             async.parallel queue, (err) ->
               return kallback err  if err
-              # transfer ownership
-              owner.update { $set: { targetId: account.getId() } }, kallback
+
+              slug ?= 'koding'
+
+              # notify group for new owner
+              contents =
+                role: 'owner'
+                id: accountId
+                group: slug
+                adminNick: owner.profile.nickname
+
+              JGroup.one { slug }, (err, group) ->
+                if group
+                  group.sendNotification 'MembershipRoleChanged', contents
+
+                ownershipRelation.update { $set: { targetId: account.getId() } }, kallback
```

Comment:
it is better to remove the first relationship & add new one with the relevant data, 
1) we might cache the relationships in the future and this might cause problems
2) created at field in the _id property would have the wrong timestamp
3) i dont remember any other place where we update the relationship